### PR TITLE
[SGP-7371] Don't use CIRCLECI_WORKSPACE_ID for percy

### PR DESCRIPTION
**Pull request by caffodian**
Will need #1 to be merged first.

### CODE REVIEW

File: percy/environment.py [hunk 1]
```diff
@@ -251,7 +251,7 @@ def commit_sha(self):
 
     @property
     def parallel_nonce(self):
-        return os.getenv('CIRCLE_WORKFLOW_WORKSPACE_ID') or os.getenv('CIRCLE_BUILD_NUM')
+        return os.getenv('CIRCLE_BUILD_NUM')
```
Comment (abendig) [inline percy/environment.py:254]: What exactly does this number represent? I'm finding "The number of the CircleCI build."  as per https://circleci.com/docs/2.0/env-vars/ a bit vague. If a jobs gets parallelized into a set of separate sub-jobs, will CIRCLE_BUILD_NUM be different for each sub-job?

**Review by abendig — COMMENTED** [review]
The test changes are consistent with the code updates. 👍 

How to tell whether it solves our problem?

File: percy/environment.py (hunk 1 above)
Comment (caffodian) [reply percy/environment.py:254]: I tested this and it seems to be the same for subjob, but does generate a new number for each rebuild.  TBH the documentation wasn't super helpful, so I resorted to rebuilding and looking at the env variables 

This is not true for WORKSPACE_ID (it does not generate a new number for each rebuild)

**Comment by caffodian** [comment]
> The test changes are consistent with the code updates. 👍

> How to tell whether it solves our problem?

Install from this commit on the Platform project.  Enable percy again.  Hit rebuild and make sure it doesn't auto fail the selenium tests

**Comment by abendig** [comment]
Hm, I never approved this ... happy to try it out in production though. :)

**Comment by caffodian** [comment]
> Hm, I never approved this ... happy to try it out in production though. :)

Oops, i saw the comment and green (of course, we don't have reviews required on this repo) and clicked merge :x

**Pull request merged**